res/css/views/rooms/_BubbleLayout.scss

```diff
@@ -0,0 +1,723 @@
+/*
+Copyright 2015, 2016 OpenMarket Ltd
+Copyright 2020 The Matrix.org Foundation C.I.C.
+Copyright 2020 Tobias Büttner <dev@spiritcroc.de>
+Copyright 2020-2021 Quirin Götz <codeworks@supercable.onl>
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+$left-gutter: 56px;
+
+.sc_BubbleLayout {
```

Comment:
Should we be using an `sc` prefix?

Reply:
In one place this will collide if not.

Reply:
**For people that weren't part of the discussion on Matrix:** After discussing we've decided it would be better to use `mx_` and avoid conflicts by some class renaming